src/slack.js

```diff
@@ -9,6 +8,7 @@ import pdf from 'html-pdf'
 import {routes as r, store} from './routes'
 import renderXML from './invoices2PohodaXML'
 import {saveInvoice} from './storage'
+import {boltApp} from './index'
```

Comment:
in our call, I said we could do import like this, but it smells because we have a kind of cyclic file dependency now - in `index.js`, we want to import from other files, but not really the other way around.
what I'd do - move `boltReceiver` and `boltApp` creation to `slack.js` and export the app from there, and consider what else would make sense to be moved. also, you can think whether a new file for something would be more appropriate